node/config/types.go

```diff
@@ -32,6 +34,14 @@ type Boost struct {
 	SectorIndexApiInfo string
 	Dealmaking         DealmakingConfig
 	DAGStore           DAGStoreConfig
+	Wallets            WalletConfig
+}
+
+type WalletConfig struct {
```

Comment:
Nit, but probably this should be called `WalletsConfig` as it has config for 2 wallets.